week3/cardgame/src/Header.jsx

```diff
@@ -0,0 +1,49 @@
+import styled from "@emotion/styled";
+import theme from "./styles/theme";
+
+const Header = ({score}) => {
+    return (
+        <>
+        <HeaderWrapper theme={theme}>
+            <TitleWrapper>동물의 숲 카드게임</TitleWrapper>
+            <TextWrapper>{score}/5</TextWrapper>
+        </HeaderWrapper>
+        </>
```

Comment:
p3: 여기서 빈 <> 태그는 아무 역할도 안하는 것 같은데, 지워주는 게 어떨까?
```suggestion
        <HeaderWrapper theme={theme}>
            <TitleWrapper>동물의 숲 카드게임</TitleWrapper>
            <TextWrapper>{score}/5</TextWrapper>
        </HeaderWrapper>
```

Reply:
오 습관적으로 적어뒀는데, 여기에선 완전 빼도 되겠네요! 꼼꼼한 리뷰 감사합니다! 👍🏻